internal/model/db.go

```diff
@@ -5,13 +5,14 @@ import (
 	"fmt"
 	"log"
 	"os"
+
+	// pg sets up postgres sql
+	_ "github.com/lib/pq"
 )
 
 var db *sql.DB
 
-// InitDB initializes DB connection
-func InitDB() {
-
+func init() {
```

Comment:
This is a "special" fn that runs at startup automatically. This lets us initialize the db right away.